src/ai/backend/client/func/service.py

```diff
@@ -269,7 +270,7 @@ async def try_start(
         """
         if service_name is None:
             faker = Faker()
-            service_name = f"bai-serve-{faker.user_name()}"
+            service_name = f"bai-serve-{faker.user_name()}"[:SESSION_NAME_MAX_LENGTH]
```

Comment:
It seems unnatural to create it with user_name and then truncate it to MAX_LENGTH.
Can't we just generate a service name that fits the MAX_LENGTH length limit from?

Reply:
The intention was to preserve the `bai-serve-` prefix. I didn’t consider `faker.user_name()` particularly meaningful in this context, as it generates a random string that isn't directly related to the actual user.